src/Sylius/Bundle/CurrencyBundle/Resources/translations/messages.fr.yml

```diff
@@ -4,11 +4,11 @@
 sylius:
     form:
         currency:
-            base: Devise de référence
-            code: Nom
-            enabled: Activée
-            exchange_rate: Taux de change
+            base: Devise de base
+            code: Code
+            enabled: Activé
+            exchange_rate: Taux de conversion
         exchange_rate:
-            source_currency: Devise d'origine
-            target_currency: Devise cible
-            ratio: Ratio
+            source_currency: Devise source
+            target_currency: Devise de destination
+            ratio: Rapport
```

Comment:
I'm not sure these changes were needed here as the original translations seemed perfectly fine.